server/src/main/java/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -2035,23 +2035,23 @@ public Pair<List<? extends Network>, Integer> searchForNetworks(ListNetworksCmd
                 if (Arrays.asList(Network.NetworkFilter.Domain, Network.NetworkFilter.AccountDomain, Network.NetworkFilter.All).contains(networkFilter)) {
                     //add domain specific networks of domain + parent domains
                     networksToReturn.addAll(listDomainSpecificNetworksByDomainPath(buildNetworkSearchCriteria(sb, keyword, id, isSystem, zoneId, guestIpType, trafficType, physicalNetworkId, networkOfferingId,
-                            aclType, skipProjectNetworks, restartRequired, specifyIpRanges, vpcId, tags, display, vlanId, associatedNetworkId), searchFilter, path, isRecursive));
+                            aclType, true, restartRequired, specifyIpRanges, vpcId, tags, display, vlanId, associatedNetworkId), searchFilter, path, isRecursive));
```

Comment:
@weizhouapache is it ok to skip the project networks in all these cases? can you check and confirm when the project networks can be skipped.

Reply:
thanks for confirming it @weizhouapache